test/components/Banners/__snapshots__/ErrorMessage.spec.js.snap

```diff
@@ -50,11 +50,11 @@ exports[`ErrorMessage component Should render correctly if two fa session expire
   <div
     className="u-flex-shrink-0 u-mb-1"
   >
-    <ComponentWithKonnectorJob
+    <Wrapper
```

Comment:
why do we have these changes in the snapshots ? Maybe a `displayName` is missing somewhere? (here: https://github.com/cozy/cozy-libs/blob/master/packages/cozy-harvest-lib/src/components/hoc/withKonnectorJob.jsx ?)

Reply:
Good point, thanks !
https://github.com/cozy/cozy-libs/pull/507

Reply:
I suggest to merge as it and update those snapshots in the next cozy-harvest-lib update. Do you agree ?

Reply:
as you want !